12LessonTwelve/in-class-project/index.html

```diff
@@ -7,8 +7,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link href="css/css-reset.css" rel="stylesheet">
-    <title></title>
-    <!-- Google font-->
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
     <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
     <link rel="stylesheet" href="css/style.css">
 </head>
```

Comment:
![screen shot 2016-09-13 at 5 38 57 pm](https://cloud.githubusercontent.com/assets/12698279/18493996/2e5536b0-79d9-11e6-8ea4-74ea6d3350bc.png)

Please make adjustments to your pricing divs in both desktop and phone sizes. Once you can get the changes made, I can bump your grade up. Overall, though, good job.